app/template/rest.py

```diff
@@ -126,6 +126,8 @@ def update_template(service_id, template_id):
         raise InvalidRequest(errors, status_code=400)
 
     update_dict = template_schema.load(updated_template).data
+    if update_dict.archived:
+        update_dict.folder = None
```

Comment:
Is this related to https://github.com/cds-snc/notification-api/pull/1565/files#diff-d3ce135e4e0dcab49ab03e15c65040fcfa1fec9dc64722849014c1f46fdc83feL44 ?

Reply:
👍